ci/test-clang-docker.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ ! -d /repo ]]; then
+    echo "[-] This script needs to be run within Docker! Use test-clang.sh instead."
+    exit 1
+fi
+
+clang=/repo/toolchain/bin/clang
+hello_c=/repo/llvm-project/hello.c
+hello_cpp=/repo/llvm-project/hello.cpp
```

Comment:
how does `repo/` get copied into the docker container? I'm guessing that's what the `:repo:ro` is doing? Some kind of shorthand rather than `docker cp`?

Reply:
I think I answered this above, let me know if you have follow up questions!